src/registrar/assets/sass/_theme/_admin.scss

```diff
@@ -847,3 +847,8 @@ div.dja__model-description{
         }
     }
 }
+
+ul.add-list-reset {
```

Comment:
May not be needed, look at add-list-reset: https://designsystem.digital.gov/utilities/list-reset/